devtools/gradle/src/main/java/io/quarkus/gradle/tasks/QuarkusDev.java

```diff
@@ -242,7 +242,11 @@ public void startDev() {
                         e);
             }
             for (AppDependency appDep : appModel.getAllDependencies()) {
-                addToClassPaths(classPathManifest, context, appDep.getArtifact().getPath().toFile());
+                if (!appDep.getArtifact().isSubModule()) {
+                    addToClassPaths(classPathManifest, context, appDep.getArtifact().getPath().toFile());
+                } else {
+                    getLogger().info("Ignoring {} as it is a project dependency", appDep);
```

Comment:
Should this one be debug instead of info?

Reply:
It could be.. but the logging of the added dependencies in addToClasspaths is info so I thought it made sense to be on same level

Reply:
+1 to be debug